Resources/Prototypes/_Mono/Research/mech_research.yml

```diff
@@ -48,8 +24,9 @@
   - ClarkeCentralElectronics
   - ClarkePeripheralsElectronics
```

Comment:
there's no reason to make ripley ever with this, we should buff ripley somehow
maybe give ripley like 4 slots and way higher punch damage